src/protocols/vnc/vnc.c

```diff
@@ -385,6 +385,17 @@ void* guac_vnc_client_thread(void* data) {
     }
 #endif
 
+    /* Disable remote console */
+    if (settings->disable_remote_input) {
+        rfbSetServerInputMsg no_remote_console = {
+                .type = rfbSetServerInput, .status = 1, .pad = 0
+        };
+        rfbBool success = WriteToRFBServer(rfb_client,
+                                           (char *) &no_remote_console, sz_rfbSetServerInputMsg);
```

Comment:
Why not leverage `PermitServerInput()`?

Reply:
I have no clue, this is based on custom code written by another developer inside our company, I have rewritten most of it so it is configurable and does not always disable the input. As requested I have changed this to PermitServerInput.